### ISSUE TITLE
Customize HTML node type that gets injected into

### ISSUE THREAD

**Issue by dfagan2** [aquibm/react-liquid#5]
Great plugin - thank you!

It would be awesome to be able to customize the DOM node that gets injected into here:
https://github.com/aquibm/react-liquid/blob/7abeb32129dea65b0e4df7ac5526f064bab2870f/src/components/ReactLiquid.js#L76

As an example, it'd be really useful for list building to designate that you'd like to dangerously set the inner HTML of a `<li>` instead of a `<div>`

So some way to pass in the container element, or signify what type to use

**Comment by aquibm** [comment]
Heya, sorry, I haven't been very active of late. I think we can resolve this and #4 by introducing a render prop.

So essentially you could do something like this:

```javascript
<ReactLiquid
    template="<p>Some {{ thing }}</p>"
    data={{  thing: "data" }}
    render={ renderedTemplate => {
        // Issue 4, call something / log / notify someone that content is available
       console.log('new template rendered')

        return (
            // Render the compiled template however you'd like
            <li dangerouslySetInnerHTML={renderedTemplate} />
        );
    }}
/>
```
I'll try to get a new version out soon :)

**Comment by aquibm** [comment]
Published as part of 1.3.0
